### PR TITLE
[12.0][MIG] website_img_dimension

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: website_img_dimension/templates/assets.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <template id="assets_editor" inherit_id="website.assets_editor">
+        <xpath expr="." position="inside">
+            <script
+                    src="/website_img_dimension/static/src/js/website_img_dimension.tooltip.js"/>
```
Comment (nikul-serpentcs) [inline website_img_dimension/templates/assets.xml]: @smurf-U change filename like `website_img_dimension_tooltip.js`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Travis fail & code check with JSlinter: https://travis-ci.org/OCA/website/jobs/476759844#L591-L602
https://travis-ci.org/OCA/website/jobs/476759844#L589

**Comment by pedrobaeza** [comment]
Please squash together the commits this way for optimizing the commit history:

seleccion_007

File: website_google_tag_manager/README.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
-==================
```
Comment (simahawk) [inline website_google_tag_manager/README.rst:1]: this has been fixed already, no?

File: website_google_tag_manager/README.rst [hunk 3]
```diff
@@ -17,13 +17,13 @@ Google Tag Manager
     :target: https://github.com/OCA/website/tree/12.0/website_google_tag_manager
     :alt: OCA/website
 .. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
-    :target: https://translation.odoo-community.org/projects/website-12-0/website-12-0-website_google_tag_manager
+    :target: https://translation.odoo-community.org/projects/website-11-0/website-11-0-website_google_tag_manager
```
Comment (simahawk) [inline website_google_tag_manager/README.rst]: in any case these are wrong

**Review by simahawk — COMMENTED** [review]
LG  apart the google tag manager change

File: website_google_tag_manager/README.rst (hunk 2 above)
Comment (smurf-U) [reply website_google_tag_manager/README.rst:1]: rebased branch...

**Review by simahawk — APPROVED** [review]

**Review by tarteo — APPROVED** [review]
LGTM (code and functionality)

**Pull request merged**